packages/@uppy/companion/package.json

```diff
@@ -89,6 +90,6 @@
     "test:watch": "npm test -- --watch"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=10.0.0"
```

Comment:
i would do this in a separate PR because it also needs a travis.yml update (and maybe docs?)

Reply:
yeah, that slipped in by accident. Thank you